### DISCUSSION TITLE
CP not unloading trailer at unload point.

### DISCUSSION THREAD

**Discussion by BRADP1980** [Courseplay/Courseplay_FS22#2517] [Q&A]
When I am using CP to harvest a field and I set my tractor and trailer to unload combine I also try to set it to go to an unload point say grain mill. He will go unload the combine, but when hitting the empty at say 90%, It only goes back to the target point on the field and then stops CP all together and never goes to unload. I have to manually take the trailer to unload. Am I missing something or doing something wrong? I know I could do it with AD, but if it works with CP that's just one less thing I have to set up. Or do I need AD for that to work correctly?

**Comment by Tensuko** [comment]
If use Giants to do this no need to have AD active.
If Giants finds the target is not on our side.
Vanilla Map?

**Reply by BRADP1980** [reply]
Do you mean no edits to the map's files then Yes Vanilla map. Griffin, Indiana. If you mean am I using other mods, then yes I am. But the map's files are untouched. Seems everything works on CP except I can't get it to go to the unloading (grain mill) after full. It goes back to target then cuts itself off. Green play button turns off.

**Reply by Tensuko** [reply]
This map?
https://farming-simulator.com/mod.php?lang=de&country=de&mod_id=252262&title=fs2022

If yes, thats not vanilla.
Vanilla Means no Mod at all.
Giants Maps!
On Mod Maps it can happen that the Giants Driver can't find a way to the sell station, wich is not CP related.
Else no Path was found or the selling station doesn't accept the fruit.
